src/sentry/demo/data_population.py

```diff
@@ -119,6 +129,8 @@
 
 mobile_platforms = ["apple-ios", "android", "react-native"]
 
+alert_platforms = ["android", "python"]
```

Comment:
@roggenkemper wonder if we can make this a property of `DataPopulation` and append to it when we call `generate_alerts`

Reply:
maybe not cause we call `generate_alerts` later now, nvm

Reply:
yeah, since we call it later and it uses it to decide which event times to track, I think it needs to be defined before.